cernopendata_client/cli.py

```diff
@@ -213,8 +214,27 @@ def get_file_locations(server, recid, doi, title, protocol, expand, verbose):
     default=False,
     help="Verify downloaded data file integrity.",
 )
+@click.option(
+    "--retry",
+    default=DOWNLOAD_RETRY_LIMIT,
+    type=click.INT,
+    help="Number of retries when downloading a file.. [Deafult={}]".format(
```

Comment:
Note the typo, please replace by `[default=...]` we use it with lowercase elsewhere in this command, see `--help`.

Reply:
BTW, now that we have two variables for retry functionality, you can perhaps expose both, so that people can also set sleeptimes:

```console
--retry-limit 10 --retry-sleep 30 
```

Reply:
There is also one more Deafult typo in the code base that you can fix at the same time:

```console
$ git grep Deaf
cernopendata_client/cli.py:221:    help="Number of retries when downloading a file.. [Deafult={}]".format(
cernopendata_client/cli.py:411:    help="Timeout for list-directory command. [Deafult={}]".format(
```

Reply:
One more thing regarding the new options: please add validation so that only positive integers are accepted.  (Kind of like for record IDs!) Currently negative values are accepted:

```console
$ cernopendata-client download-files --recid 3005 --verify --retry-limit -1 --retry-sleep -10
==> Downloading file 1 of 1
  -> File: ./3005/0d0714743f0204ed3c0144941e6ce248.configFile.py
  -> Progress: 3/3 kiB (100%)
==> Verifying file 0d0714743f0204ed3c0144941e6ce248.configFile.py...
  -> Expected size 3644, found 3644
  -> Expected checksum adler32:be83a186, found adler32:be83a186
==> Success!
```